cmd/torrent-verify/main.go

```diff
@@ -16,16 +16,40 @@ import (
 )
 
 var (
-	filePath = flag.String("torrent", "/path/to/the.torrent", "path of the torrent file")
-	dirPath  = flag.String("path", "/torrent/data", "path of the torrent data")
+	torrentPath = flag.String("torrent", "/path/to/the.torrent", "path of the torrent file")
+	dataPath    = flag.String("path", "/torrent/data", "path of the torrent data")
+	summary     = flag.Bool("summary", false, "display summary at the end")
 )
 
-func init() {
-	flag.Parse()
+func verifySummary(sMap map[bool][]int) {
+	fmt.Println("----------------")
+	fmt.Println(" TORRENT-VERIFY ")
+	fmt.Println("----------------")
+	fmt.Printf("Number of correct pieces: %d\n", len(sMap[true]))
+	fmt.Printf("Number of wrong pieces: %d\n", len(sMap[false]))
```

Comment:
The reason why I added the summary flag is, if you have a file with thousands of pieces they're logged in the ```stdout``` and unless you parse them with some UNIX cli tool magic it can get pretty verbose. 
The new ```summary``` flag is optional and just gives a quick overview only if you wish to see one. I'm happy to remove it completely.